doc/contributing/maintaining-V8.md

```diff
@@ -208,6 +208,62 @@ backport the fix:
 Abandoned V8 branches are supported in the Node.js repository. The fix needs
 to be cherry-picked in the Node.js repository and V8-CI must test the change.
 
+As an example for how to backport changes, consider the bug [RegExp show
+inconsistent result with other browsers](https://crbug.com/v8/5199). From the
+bug we can see that it was merged by V8 into 5.2 and 5.3, and not into V8 5.1
+(since it was already abandoned). Since Node.js `v6.x` uses V8 5.1, the fix
+needed to be backported.
+
+#### Backporting with `git-node` (recommended)
+
+You can use [`git-node`][] to help you backport patches. This removes
+some manual steps and is recommended.
+
+Here are the steps for the bug mentioned above:
+
+1. Install `git-node` by installing [`node-core-utils`][].
+2. Install the prerequisites for [`git-node-v8`][].
+3. Find the commit hash linked-to in the issue (in this case a51f429).
+4. Checkout a branch off the appropriate _vY.x-staging_ branch (e.g.
+   _v6.x-staging_ to fix an issue in V8 5.1).
+5. Run `git node v8 backport a51f429`.
+6. If there are conflicts, `git-node` will wait for you to resolve them.
+   `git-node` will prompt you to resolve them in another terminal and then
+   enter `RESOLVED`:
+
+```console
+$ git node v8 backport a51f429
+✔ Update local V8 clone
+❯ V8 commit backport
+  ✔ Get current V8 version
+  ✔ Generate patches
+  ❯ Apply and commit patches to deps/v8
+    ❯ Commit a51f429772d1
+      ⠏ Apply patch
+      ◼ Increment embedder version number
+      ◼ Commit patch
+
+? Resolve merge conflicts and enter 'RESOLVED' ‣
+```
+
+7. After you resolve conflicts (or if there are no conflicts), the
+   output should look like this:
+
+```console
+$ git node v8 backport a51f429
+✔ Update local V8 clone
+✔ V8 commit backport
+```
+
+8. Open a PR against the v6.x-staging branch in the Node.js repository.
+   Launch the normal and [V8 CI][] using the Node.js CI system. We only
+   needed to backport to v6.x as the other LTS branches weren't affected
+   by this bug.
+
+See [`git-node-v8-backport`][] for more documentation and additional options.
+
+#### Backporting manually (not recommended)
```

Comment:
```suggestion
#### Backporting manually
```

Reply:
I also removed "(not recommended)" from the other header I added below. Do you think that the "(recommended)" part should stay; or should I remove those too?

Reply:
The "(recommended)" bit is OK (unless someone else thinks there's a reason not to recommend it?), I just don't think it's necessary to insist that the manual process is "not recommended" (I would reserve that wording for something that's dangerous or can have unforeseen consequences).